file-formats/sajc/examples/z_aff_example_sajc.sajc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "in non elit",
+    "originalLanguage": "fj",
+    "abapLanguageVersion": "standard"
```

Comment:
@wurzka is this an error from the generation tool?

Reply:
No, it should be removed here. The ST wouldn't write it to the JSON data file

Reply:
done in [641b254](https://github.com/SAP/abap-file-formats/pull/313/commits/641b25443b1c2dc68b995c20eb36868e92f3aeec)